docs/recipes/make-stream-from-buffer.md

```diff
@@ -79,13 +79,14 @@ gulp.task('write-versions', function() {
   availableVersions.forEach(function(v) {
     // make a new stream with fake file name
     var stream = source('final.' + v);
+    
+    var streamEnd = stream;
```

Comment:
This assignment is superfluous, isn't it? You could just declare `streamEnd`, or eschew it altogether and just do `stream = stream ...` on line 98.